[16:40:16] [DEBUG] Running client sanity checks...
[16:40:16] [DEBUG] Loaded Extension: interactions.ext.jurigged
[16:40:17] [DEBUG] No setup function found in play
[16:40:17] [DEBUG] Found extension class Music in play: Attempting to load
[16:40:17] [DEBUG] Added callback: Music._menu
[16:40:17] [DEBUG] Added callback: Music._pause
[16:40:17] [DEBUG] Added callback: Music._resume
[16:40:17] [DEBUG] Added callback: Music._skip
[16:40:17] [DEBUG] Added callback: Music._stop
[16:40:17] [DEBUG] Added callback: Music.play
[16:40:17] [DEBUG] Loaded Extension: play
[16:40:18] [DEBUG] No setup function found in askbard
[16:40:18] [DEBUG] Found extension class Bard in askbard: Attempting to load
[16:40:18] [DEBUG] Added callback: Bard._askbard
[16:40:18] [DEBUG] Added callback: Bard._endbard
[16:40:18] [DEBUG] Added callback: Bard.set_bard_channel
[16:40:18] [DEBUG] Loaded Extension: askbard
[16:40:18] [DEBUG] No setup function found in noi_chu
[16:40:18] [DEBUG] Found extension class NoiChu in noi_chu: Attempting to load
[16:40:18] [DEBUG] Added callback: NoiChu._reset
[16:40:18] [DEBUG] Added callback: NoiChu._world_setup
[16:40:18] [DEBUG] Loaded Extension: noi_chu
[16:40:18] [DEBUG] No setup function found in db_refesh
[16:40:18] [DEBUG] Found extension class Database in db_refesh: Attempting to load
[16:40:18] [DEBUG] Loaded Extension: db_refesh
[16:40:18] [DEBUG] No setup function found in help
[16:40:18] [DEBUG] Found extension class Help in help: Attempting to load
[16:40:18] [DEBUG] Added callback: Help._about
[16:40:18] [DEBUG] Added callback: Help.help
[16:40:18] [DEBUG] Loaded Extension: help
[16:40:18] [DEBUG] No setup function found in channel_listen
[16:40:18] [DEBUG] Found extension class ChannelListen in channel_listen: Attempting to load
[16:40:18] [DEBUG] Added callback: ChannelListen._setup
[16:40:18] [DEBUG] Loaded Extension: channel_listen
[16:40:18] [DEBUG] Added callback: on_start
[16:40:18] [DEBUG] 1 callbacks have been loaded from __main__.
[16:40:18] [DEBUG] Added callback: _disconnect
[16:40:18] [DEBUG] Added callback: _on_websocket_ready
[16:40:18] [DEBUG] Added callback: on_autocomplete_completion
[16:40:18] [DEBUG] Added callback: on_autocomplete_error
[16:40:18] [DEBUG] Added callback: on_command_completion
[16:40:18] [DEBUG] Added callback: on_command_error
[16:40:18] [DEBUG] Added callback: on_component_completion
[16:40:18] [DEBUG] Added callback: on_component_error
[16:40:18] [DEBUG] Added callback: on_error
[16:40:18] [DEBUG] Added callback: on_modal_completion
[16:40:18] [DEBUG] Added callback: on_modal_error
[16:40:18] [DEBUG] Added callback: on_resume
[16:40:18] [DEBUG] 12 callbacks have been loaded from Client.
[16:40:18] [DEBUG] Attempting to login
[16:40:19] [DEBUG] Caching ingested rate limit data for: 78bb8553d9352a5a2f89f9def401287a
[16:40:19] [DEBUG] GET /users/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:40:19] [DEBUG] GET /users/@me Received 200 :: [0/1000 calls remaining]
[16:40:19] [DEBUG] Caching ingested rate limit data for: d28a133af187e91d71d96a223467ce49
[16:40:19] [DEBUG] GET /oauth2/applications/@me Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:40:19] [DEBUG] GET /oauth2/applications/@me Received 200 :: [0/1000 calls remaining]
[16:40:19] [WARNING] Setting sync_ext to True by default for syncing changes
[16:40:19] [INFO] Loading jurigged
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\Main.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\play.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\embed.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\askbard.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\noi_chu.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\db_refesh.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\help.py
[16:40:19] [DEBUG] Watch D:\Project\Python\qpneruy-Git\music\channel_listen.py
[16:40:19] [DEBUG] Caching ingested rate limit data for: 0a6d458104b103c7be293cc9b284d9ee
[16:40:19] [DEBUG] GET /gateway Received 200 :: [99/100 calls remaining]
[16:40:19] [INFO] Shard ID 0 | Starting Shard
[16:40:19] [INFO] Shard ID 0 | Shard is attempting to connect to gateway...
[16:40:20] [DEBUG] Shard ID 0 | Identification payload sent to gateway, requesting intents: 3407613
[16:40:21] [INFO] Shard ID 0 | Gateway connection established
[16:40:21] [DEBUG] Shard ID 0 | Session ID: c1c9bc4dde6884e73331432281de459f Trace: ['["gateway-prd-us-east1-b-492s",{"micros":1441671,"calls":["id_created",{"micros":969,"calls":[]},"session_lookup_time",{"micros":2267,"calls":[]},"session_lookup_finished",{"micros":15,"calls":[]},"discord-sessions-prd-2-115",{"micros":1436903,"calls":["start_session",{"micros":1348246,"calls":["discord-api-6c869fb97d-hksjp",{"micros":172500,"calls":["get_user",{"micros":41217},"get_guilds",{"micros":19021},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":2},"authorized_ip_coro",{"micros":12}]}]},"starting_guild_connect",{"micros":86,"calls":[]},"presence_started",{"micros":338,"calls":[]},"guilds_started",{"micros":152,"calls":[]},"guilds_connect",{"micros":2,"calls":[]},"presence_connect",{"micros":88050,"calls":[]},"connect_finished",{"micros":88055,"calls":[]},"build_ready",{"micros":22,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":1,"calls":[]},"split_ready",{"micros":0,"calls":[]}]}]}]']
[16:40:21] [DEBUG] Dispatching Event: websocket_ready
[16:40:22] [DEBUG] Dispatching Event: guild_join
[16:40:22] [DEBUG] Dispatching Event: guild_join
[16:40:22] [DEBUG] Dispatching Event: guild_join
[16:40:22] [DEBUG] Dispatching Event: guild_join
[16:40:22] [DEBUG] Dispatching Event: guild_join
[16:40:22] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:40:22] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:40:22] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:40:22] [DEBUG] Waiting for bucket dd84d9818020c27975c0d5ab4a794d44 to unlock.
[16:40:22] [DEBUG] Caching ingested rate limit data for: dd84d9818020c27975c0d5ab4a794d44
[16:40:22] [DEBUG] GET /applications/{application_id}/commands Has exhausted its ratelimit (1000)! Locking route for 0.001 seconds
[16:40:22] [DEBUG] GET /applications/{application_id}/commands Received 200 :: [0/1000 calls remaining]
[16:40:22] [DEBUG] 0 is already up-to-date with 14 commands.
[16:40:22] [DEBUG] Sync of 1 scopes took 0.7467823000042699 seconds
[16:40:22] [DEBUG] Dispatching Event: startup
[16:40:22] [DEBUG] Sending data to websocket: {"op": 3, "d": {"since": 1703756422805, "activities": [{"name": "/help for helpful", "type": 0}], "status": "idle", "afk": false}}
[16:40:29] [DEBUG] Sending heartbeat every 41.25 seconds
[16:40:29] [DEBUG] Sending data to websocket: {"op": 1, "d": 6}
[16:40:29] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:40:29] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:40:34] [DEBUG] 1174695146214928500::play should be called
[16:40:34] [DEBUG] Dispatching Event: message_create
[16:40:34] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:40:34] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NTQ2OTk3MzM3MzAxODpmcDlIOUFQZWtUV2ZWNDU4R3VkVjc2YXpGZEpkSWxITThMSlhDZlBnU01OUnFxVERxc0RnYzQ1RE83YzhyaUtPMXJVaG5XV1FGcVIweWVpc0UxeEVuaXRxNDNLcG1LSEc4d2JqdXlzWFVqUHRIZXZyMjMyYThlQjBpR2NNaWVBYQ/callback Received 204 :: [4/5 calls remaining]
[16:40:34] [DEBUG] Sending data to websocket: {"op": 4, "d": {"guild_id": 1067024293709496340, "channel_id": 1189851990356918383, "self_mute": false, "self_deaf": false}}
[16:40:34] [DEBUG] Waiting for voice connection data...
[16:40:34] [DEBUG] Dispatching Event: voice_user_join
[16:40:35] [DEBUG] Attempting to initialise voice gateway...
[16:40:36] [DEBUG] Voice Connection has identified itself to Voice Gateway
[16:40:36] [DEBUG] Discord send VC Ready! Establishing a socket connection...
[16:40:36] [DEBUG] IP Discovery in progress...
[16:40:36] [DEBUG] Voice Initial Response Received: b'\x00\x02\x00F\x00\x05cC116.100.183.123\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xcb/'
[16:40:36] [DEBUG] IP Discovered: 116.100.183.123 #52015
[16:40:36] [DEBUG] Sending data to websocket: {"op": 1, "d": {"protocol": "udp", "data": {"address": "116.100.183.123", "port": 52015, "mode": "xsalsa20_poly1305_lite"}}}
[16:40:36] [DEBUG] Starting UDP Keep Alive
[16:40:36] [DEBUG] Unhandled OPCODE: 18 = data = {'user_id': '610841389164396565', 'flags': 2}
[16:40:36] [DEBUG] Unhandled OPCODE: 20 = data = {'user_id': '610841389164396565', 'platform': 0}
[16:40:36] [INFO] Voice connection established; using xsalsa20_poly1305_lite
[16:40:36] [DEBUG] Dispatching Event: command_completion
[16:40:36] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:40:38] [DEBUG] Sending heartbeat every 13.75 seconds
[16:40:38] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.24736175816698203}
[16:40:38] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:40:38] [DEBUG] ❤ Heartbeat acknowledged after 0.08475 seconds
[16:40:51] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.24685269708181856}
[16:40:51] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:40:51] [DEBUG] ❤ Heartbeat acknowledged after 0.08682 seconds
[16:41:05] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.41932358645603174}
[16:41:05] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:41:05] [DEBUG] ❤ Heartbeat acknowledged after 0.09086 seconds
[16:41:10] [DEBUG] Sending data to websocket: {"op": 1, "d": 10}
[16:41:10] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:41:10] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:41:19] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.9053634697378338}
[16:41:19] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:41:19] [DEBUG] ❤ Heartbeat acknowledged after 0.08536 seconds
[16:41:33] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.26298057109595785}
[16:41:33] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:41:33] [DEBUG] ❤ Heartbeat acknowledged after 0.12353 seconds
[16:41:39] [DEBUG] Caching play
[16:41:39] [DEBUG] Update play.Music.play:93
[16:41:39] [DEBUG] Update play.Music.play:93
[16:41:39] [DEBUG] Update play.Music:69
[16:41:39] [DEBUG] Update play.Music:69
[16:41:39] [DEBUG] Update play:1
[16:41:39] [DEBUG] Update play:1
[16:41:39] [DEBUG] Checking play
[16:41:39] [DEBUG] Checking for changes
[16:41:39] [DEBUG] No changes detected
[16:41:46] [DEBUG] 1174695146214928500::play should be called
[16:41:46] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.6473199410984238}
[16:41:46] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:41:46] [DEBUG] ❤ Heartbeat acknowledged after 0.17979 seconds
[16:41:47] [DEBUG] Dispatching Event: message_create
[16:41:47] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:41:47] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NTc3MjMzMjM2MzgxNzpRZlUyaU96WElJOUdnVGhsR1RNc011d21idUwyM29sN0JpS0VyMWJWY1NNU2ZGbDBHSWlOWjRJWnowT1ZyMG9RbVRXaGRhSHN4UFBsc3RoUlBiSExWUFZkaGE5amxZR0RXOE4xTllLeEdobU5tWUFHVEs1eldRdlRBc2NiSVFtSw/callback Received 204 :: [4/5 calls remaining]
[16:41:48] [DEBUG] Dispatching Event: command_error
[16:41:48] [DEBUG] Dispatching Event: command_completion
[16:41:48] [DEBUG] Dispatching Event: error
[16:41:48] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:41:48] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 123, in play
    elif "https://www.youtube.com/playlist?list=" in song:
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 37, in playlist_process
    url = url_list.pop()
          ^^^^^^^^^^^^^^
TypeError: pop expected at least 1 argument, got 0

[16:41:51] [DEBUG] Sending data to websocket: {"op": 1, "d": 12}
[16:41:51] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:41:52] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:42:00] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.8430943268366147}
[16:42:00] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:42:00] [DEBUG] ❤ Heartbeat acknowledged after 0.09159 seconds
[16:42:14] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.3501309565258336}
[16:42:14] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:42:14] [DEBUG] ❤ Heartbeat acknowledged after 0.08610 seconds
[16:42:28] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.3950817536620509}
[16:42:28] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:42:28] [DEBUG] ❤ Heartbeat acknowledged after 0.08960 seconds
[16:42:33] [DEBUG] Sending data to websocket: {"op": 1, "d": 12}
[16:42:33] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:42:33] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:42:41] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.29002487961950973}
[16:42:41] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:42:41] [DEBUG] ❤ Heartbeat acknowledged after 0.09272 seconds
[16:42:55] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.9036396219725509}
[16:42:55] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:42:55] [DEBUG] ❤ Heartbeat acknowledged after 0.12175 seconds
[16:43:09] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.44963315372584955}
[16:43:09] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:43:09] [DEBUG] ❤ Heartbeat acknowledged after 0.08671 seconds
[16:43:14] [DEBUG] Sending data to websocket: {"op": 1, "d": 12}
[16:43:14] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:43:14] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:43:21] [DEBUG] Caching play
[16:43:21] [DEBUG] Run play:3 | from collections import deque
[16:43:21] [DEBUG] Update play.Music.play:93
[16:43:21] [DEBUG] Update play.Music:69
[16:43:21] [DEBUG] Update play:1
[16:43:21] [DEBUG] Checking play
[16:43:21] [DEBUG] Checking for changes
[16:43:21] [DEBUG] No changes detected
[16:43:23] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.6971858622070762}
[16:43:23] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:43:23] [DEBUG] ❤ Heartbeat acknowledged after 0.08823 seconds
[16:43:24] [DEBUG] 1174695146214928500::play should be called
[16:43:24] [DEBUG] Dispatching Event: message_create
[16:43:24] [DEBUG] Caching ingested rate limit data for: 432f6223265d156004ac81366c5a948f
[16:43:24] [DEBUG] POST /interactions/{interaction_id}/aW50ZXJhY3Rpb246MTE4OTg2NjE4MzA1OTU4NzA4Mjp4eWNNb29zdFh1NE5KN1dMNHlCanZndm5iMG8yUzREb1ZBVzNuZzF3WUlaMXA3dXFHbWZzTENESUFjM1JhNmhMbjU4V3hIRXFxSE84dW9LM3BrVDNIZWVabExiMmdFTXVIdHpaMWZ1VXV3MjJOU1NyaTFOeXg1aW14NEhYUHlGZA/callback Received 204 :: [4/5 calls remaining]
[16:43:25] [DEBUG] Dispatching Event: command_error
[16:43:25] [DEBUG] Dispatching Event: command_completion
[16:43:25] [DEBUG] Dispatching Event: error
[16:43:25] [INFO] Command Called: play with event.ctx.args = ['https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'] | event.ctx.kwargs = {'song': 'https://www.youtube.com/playlist?list=PLL-LXY9dtjAYgBX0TiNDwAbgThsGBb8jD'}
[16:43:25] [ERROR] Ignoring exception in cmd `/play`:
Traceback (most recent call last):
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1900, in __dispatch_interaction
    response = await callback
               ^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\client.py", line 1771, in _run_slash_command
    return await command(ctx, **ctx.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\command.py", line 132, in __call__
    await self.call_callback(self.callback, context)
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\application_commands.py", line 833, in call_callback
    return await self.call_with_binding(callback, ctx, *new_args, **new_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huuti\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\models\internal\callback.py", line 43, in call_with_binding
    return await callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\music\play.py", line 123, in play
    await ctx.send(embeds=embed, components=[self.hang1, self.hang2])
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Project\Python\qpneruy-Git\modules\Music_queue.py", line 37, in playlist_process
    url = url_list.pop()
          ^^^^^^^^^^^^^^
TypeError: pop expected at least 1 argument, got 0

[16:43:36] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.8997157227149498}
[16:43:36] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:43:36] [DEBUG] ❤ Heartbeat acknowledged after 0.08615 seconds
[16:43:50] [DEBUG] Sending data to websocket: {"op": 3, "d": 0.05196734327668495}
[16:43:50] [DEBUG] ❤ Voice Connection is sending Heartbeat
[16:43:50] [DEBUG] ❤ Heartbeat acknowledged after 0.11702 seconds
[16:43:55] [DEBUG] Sending data to websocket: {"op": 1, "d": 14}
[16:43:55] [DEBUG] Shard ID 0 | ❤ Gateway is sending a Heartbeat
[16:43:56] [DEBUG] Shard ID 0 | ❤ Received heartbeat acknowledgement from gateway
[16:44:02] [DEBUG] Dispatching Event: disconnect
[16:44:02] [DEBUG] Stopping the bot.
[16:44:02] [INFO] Shard ID 0 | Stopping Shard
